### DISCUSSION TITLE
Zigbee2mqtt not working anymore :-(

### DISCUSSION THREAD

**Discussion by Smokey1999** [Koenkk/zigbee2mqtt#24270] [General]
Hello everyone,

I am running Home Assistant OS on a virtual machine (VM) and using the Zigbee2MQTT add-on. Recently, Zigbee2MQTT suddenly stopped working, even though my USB adapter is still being detected. I didn’t restart the VM or make any significant changes before this issue occurred.

Here are the relevant version details:

Core: 2024.10.1
Supervisor: 2024.10.0
Operating System: 13.1
Zigbee2MQTT version: 1.40.2
Zigbee-herdsman version: 2.1.3
Mosquitto broker: 6.4.1
The device is connected, and my configuration seems correct, as you can see in the attached screenshots of my settings. The device path is:
/dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_b6acd18d9045ed11820ac58f0a86e0b4-if00-port0.

I've confirmed that the virtual machine is detecting the USB dongle properly, and my Mosquitto broker is running without issues, but Zigbee2MQTT still fails to start. The error message I’m seeing is:

```
Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
I’ve checked permissions, reviewed the documentation, and restarted the Mosquitto broker, but I can’t seem to resolve the issue.
```

Has anyone experienced something similar or can point me in the right direction? Any help is greatly appreciated!

Thanks in advance!

Screenshot:
![Schermafbeelding 2024-10-08 224702](https://github.com/user-attachments/assets/67c240b5-4959-4c02-9d12-b045f3daebd4)

**Comment by lux73** [comment]
https://github.com/Koenkk/zigbee2mqtt/discussions/24199

**Reply by Smokey1999** [reply]
Thank you, I will look into that.

**Reply by Smokey1999** [reply]
@lux73 update; its working now ;-)

**Reply by lux73** [reply]
nice 🙂

**Reply by NoLooseEnds** [reply]
> @lux73 update; its working now ;-)

What did you do specifically?
